tools/admin/wskadmin

```diff
@@ -105,6 +105,12 @@ def parseArgs():
     subcmd = subparser.add_parser('whois', help='identify user from an authorization key')
     subcmd.add_argument('authkey', help='the credentials to look up')
 
+    subcmd = subparser.add_parser('block', help='block a user')
+    subcmd.add_argument('subject', help='the user to block')
```

Comment:
should this utility allow blocking per namespace? to block a namespace lookup the subject then block the corresponding document?